src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const listOfEmployees = document.querySelectorAll('li[data-position]');
+
+const convert = string => +string.replace(/[^\d.]/g, '');
+
+function sortList(list) {
+  [...list]
+    .sort((a, b) => convert(b.dataset.salary) - convert(a.dataset.salary))
+    .forEach(item => item.parentElement.appendChild(item));
+};
+
+function getEmployees(list) {
+  return [...list].map(employee => ({
+    name: employee.textContent.trim(),
```

Comment:
here you could use `.innerText` to avoid `trim()`

Reply:
ty